packages/web-config-server/src/models/AccessRequest.js

```diff
@@ -0,0 +1,25 @@
+/**
+ * Tupaia
+ * Copyright (c) 2017 - 2020 Beyond Essential Systems Pty Ltd
+ */
+
+import { TYPES } from '@tupaia/database';
+import { BaseModel } from './BaseModel';
+
+export class AccessRequest extends BaseModel {
```

Comment:
@edmofro Added AccessRequest model

Reply:
Cool, thanks for the tag

Reply:
@nick-beyondessential now that I've merged https://github.com/beyondessential/tupaia/pull/1484, this will need to be updated to live in @tupaia/database